insights/config/specs.py

```diff
@@ -80,6 +81,8 @@
     "docker_storage"            : DockerHostSimpleFileSpec("etc/sysconfig/docker-storage"),
     "docker_storage_setup"      : DockerHostSimpleFileSpec("etc/sysconfig/docker-storage-setup"),
     "docker_sysconfig"          : DockerHostSimpleFileSpec("etc/sysconfig/docker"),
+    "du_-h"                     : CommandSpec("/usr/bin/du -h /"),
+    "du_-sh"                    : CommandSpec("/usr/bin/du -sh /"),
```

Comment:
No need of this

Reply:
or if need specific to mongodb please add it.

Reply:
ok ,I'll remove `du` entries from config/specs.py file, actually from forman-debug archieve, we need to parse content of files"mongodb_disk_space" and "postgres_disk_space" which contain output of du in it (e.g, du -sh /var/lib/mongodb), i'll replace the du command specs with file specs.

Reply:
I have fixed this.